io.openems.edge.batteryinverter.deye/src/io/openems/edge/deye/common/Config.java

```diff
@@ -0,0 +1,54 @@
+package io.openems.edge.deye.common;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition(//
+		name = "Deye.Sun.Hybrid - MaMoTec", //
+		description = "Deye.Sun.Hybrid.")
+@interface Config {
+
+	@AttributeDefinition(name = "Component-ID", description = "Unique ID of this Component")
+	String id() default "ess0";
+
+	@AttributeDefinition(name = "Alias", description = "Human-readable name of this Component; defaults to Component-ID")
+	String alias() default "";
+
+	@AttributeDefinition(name = "Is enabled?", description = "Is this Component enabled?")
+	boolean enabled() default true;
+
+	@AttributeDefinition(name = "Read-Only mode", description = "Enables Read-Only mode")
+	boolean readOnlyMode() default false;
+
+	@AttributeDefinition(name = "Modbus-ID", description = "ID of Modbus bridge.")
+	String modbus_id() default "modbus0";
+
+	@AttributeDefinition(name = "Modbus-Unit-ID", description = "Unit ID of Modbus bridge.")
+	int unit_id() default 0;
+
+	@AttributeDefinition(name = "Power limit on PowerDecreaseCausedByOvertemperature error; '0' to disable power limit logic", description = "")
```

Comment:
What does this Config Point do?

Maybe consider changing it to something like:

`@AttributeDefinition(name = "ThermalPowerLimitControl", description = "Defines the power limit adjustment in response to overtemperature events. Set to '0' to disable.")
`

Reply:
Done.